### PR TITLE
Naive bayes

### PR DESCRIPTION
**Pull request by b-sherman**
I just created a new branch for the updated naive bayes files since the old one is very far behind now. Included are the images, code, and module text.

### CODE REVIEW

File: docs/source/content/supervised/bayes.rst [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+separated them into two general cases based on what type of feature
+distributions they use: continuous or discrete. Continuous means real-valued
+(you can have decimal answers) and discrete means a count (you can only have
+whole number answers). Also provided are the relevant code snippets for each
+algorithm.
+
+Gaussian Model (Continuous)
+===========================
+Gaussian models assume features follow a normal distribution. As far as you
+need to know, a normal distribution is just a specific type of distribution.
```
Comment (mergu) [inline docs/source/content/supervised/bayes.rst]: We should let the user know briefly what each of these categories are. For instance, a gaussian distribution is one where most of the values fall towards the mean (maybe have a picture too)

**Review by mergu — COMMENTED** [review]
This is very well written! I just had one suggestion

**Comment by b-sherman** [comment]
I tried to add some more graphics to break up the text and apply the review comments.

**Review by mergu — APPROVED** [review]
This looks great Brendan. Thanks for the work you put in

**Pull request merged**